librz/core/cmd/cmd_debug.c

```diff
@@ -2474,14 +2476,14 @@ RZ_IPI int rz_cmd_debug(void *data, const char *input) {
 							rz_cons_printf("%s\n", uni);
 							free(uni);
 						} else {
-							eprintf("Cannot open those alias files\n");
+							RZ_LOG_ERROR("core: Cannot open those alias files\n");
 						}
 						free(a);
 						free(b);
 					}
 					free(arg);
 				} else {
-					eprintf("Usage: dif $a $b  # diff two alias files\n");
+					RZ_LOG_ERROR("core: Usage: dif $a $b  # diff two alias files\n");
```

Comment:
`dif/diff`?

Reply:
it is the command name

Reply:
Yes, I figured. Looking up at the code, it seems both `dif` and `diff` are valid commands.